lib/ui/home/tabs/watches_tab.dart

```diff
@@ -208,41 +213,37 @@ class MyWatchesTab extends HookWidget implements CobbleScreen {
     return CobbleScaffold.tab(
       title: "My Watches",
       child: ListView(children: <Widget>[
-        Offstage(
-            offstage: isConnected,
-            child: Column(children: <Widget>[
-              Container(
-                  child: Row(children: <Widget>[
-                    Container(
-                      child: Center(
-                          child: Icon(RebbleIcons.disconnect_from_watch,
-                              color: Colors.black)),
-                      width: 56,
-                      height: 56,
-                      decoration: BoxDecoration(
-                          color: _disconnectedColor, shape: BoxShape.circle),
-                    ),
-                    SizedBox(width: 16),
-                    Column(
-                      children: <Widget>[
-                        Text("Nothing connected",
-                            style: TextStyle(fontSize: 16)),
-                        SizedBox(height: 4),
-                        Text("Background service stopped"),
```

Comment:
Hm, should we keep that text? It could be comforting to the user knowing that app does nothing in the background when no watch is connected.

Reply:
@piggehperson What do you think?

Reply:
Yeah, that was the idea behind having it there. _Does_ the app actually do nothing though? I'd like that to hide the persistent notification, for example

Reply:
Yes, app does nothing when all watches are disconnected.

Reply:
@matejdro I removed it in response to the issue you opened "top part should not say background service stopped"

Reply:
Yeah sorry I was not clear enough. I meant that it should not say service stopped, when watch was connecting."

Reply:
No worries. I'll revert that part of the change.